Projects/Team3/Andrii Nazaruk/HomeWork4.3/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace HomeWork4._3
+{
+    class Program
+    {
+        //3. Написати програму, яка знаходить в масиві значення, що повторюються два і більше разів, і показує їх на екран.
+        static void Main(string[] args)
+        {
+            Random r = new Random();
+            int[] array = new int[1000];
+            int count = 0;         
+            int repeatDigit = 0;
+
+            for(int i = 0; i < array.Length; i++)
+            {
+                array[i] = r.Next(0, 1000);
+                Console.WriteLine(array[i]);
+            }
+
+            for (int i = 1; i < array.Length; i++)
+            {
+
+                if (array[i] == array[i - 1])
+                {
+                    repeatDigit = array[i];
+                    count++;
+                    Console.WriteLine("Repeat numbers :" + repeatDigit);
+
+                }
+            }
+           
+            Console.WriteLine("Count of repeats : " + count);
+
```

Comment:
do your program able to count repeats for example in this case : 1,2,3,4,5,1,2,3
and what result will you get?

Reply:
no, I checked. I will do it again